### ISSUE TITLE
Accessibility Improvements 

### ISSUE THREAD

**Issue by paul-qg** [AvdLee/RocketSimApp#442]
List out the accessibility information that has been set up for the elements on screen with the ability to run voice over for the screen

**Comment by AgapovOne** [comment]
It would be a game changer for devs who want to support accessibility.

Reveal [already did that](https://revealapp.com/how-reveal-helps/accessibility/), but it costs much more than RocketSim.

Default accessibility inspector is too bad for debugging UI tests and supporting accessibility

**Comment by AvdLee** [comment]
@AgapovOne I've had a breakthrough for this potentially. To understand your needs: are you looking for the voice over runner specifically, or just the accessibility information? 

Like, what would be key features you need here? The more details you can give me, the better I understand your needs. 🙏

**Comment by AgapovOne** [comment]
Yes, great to hear from you :)

I have two reasons to look into accessibility hierarchy:

- Adjusting actual accessibility values like labels, traits, and other options. To see how a user with VoiceOver/VoiceControl would interact with an app. I look into grouping, values, frame, and order of an item.
- UI Testing. Whenever some element is accessed it should have a proper id, frame, traits and type, so that code inside ui test can access this element. Sometimes there are issues in accessibility containers when some group doesn't allow access into elements inside those containers. It might be good insight to know if we should group/ungroup elements or make another query.

Also you can try Reveal for free for 14 days to be inspired :)
